x-pack/plugins/security_solution/public/app/home/global_header/index.test.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
I think that we should only create new test files using `react test utils`. (not sure)